<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Start">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>启动</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeReduceURL">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>150</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>更改</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>59</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>掉粉音乐</string>
    </property>
   </widget>
   <widget class="QLabel" name="AppSecret">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>AppSecret</string>
    </property>
   </widget>
   <widget class="QLabel" name="AppID">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>59</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>AppID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Stop">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>250</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeAddURL">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>190</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>更改</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>59</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>增粉音乐</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ReduceMusicURL">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="AddMusicURL">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="AppSecretText">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="AppIdText">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="AddFans">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>增粉</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ReduceFans">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>掉粉</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ReduceCheck">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>160</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>提示</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="AddCheck">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>200</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>提示</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StopMusic">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>250</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>停止音乐</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearAppId">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearAppSecret">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>90</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
